packages/devtools_app/lib/src/inspector/flutter/story_of_your_layout/flex.dart

```diff
@@ -744,6 +637,119 @@ class _StoryOfYourFlexWidgetState extends State<StoryOfYourFlexWidget>
       ),
     );
   }
+
+  Widget _buildLayout(BuildContext context, BoxConstraints constraints) {
```

Comment:
@albertusangga, @jacob314, @kenzieschmoll, @terrylucas this is an example of when and how we want to split large widget trees up.

As a rule of thumb, if a widget tree goes over 50 lines, it's time to split it up.

This will make the code more readable and reduce the number of merge conflicts when two people work on the same file.